doc/platforms/cray.rst

```diff
@@ -460,24 +455,7 @@ Using the ugni Communications Layer
 
 To use ugni communications:
 
-1) Make sure that you are using either the GNU, Intel, or Cray target
-   compiler::
-
-     module load PrgEnv-gnu
-
-   or::
-
-     module load PrgEnv-intel
-
-   or::
-
-     module load PrgEnv-cray
-
-   (If you have a different PrgEnv module loaded, you will have to
-   unload it first, or do a swap instead of a load.)
-
-
-2) Set your CHPL_COMM environment variable to ``ugni``:
+1) Set your CHPL_COMM environment variable to ``ugni`` (the default):
```

Comment:
Eh, think I'll leave this alone for now since it matches what we say in other places. Think this is another thing that's worth simplifying once muxed is retired.